data-plane/receiver/pom.xml

```diff
@@ -35,6 +35,11 @@
       <version>${project.version}</version>
     </dependency>
 
+    <dependency>
```

Comment:
@brunobat @tsegismont this gets me locally around the `ClassNotFoundException`.

Let's see what the CI says :sweat_smile: